api.bs

```diff
@@ -431,12 +431,22 @@ which could otherwise be used to exceed [=privacy budgets=].
 
 ## Navigator Interface ## {#navigator-interface}
 
+All functionality for this API is attached
+to the `{{Navigator|navigator}}.{{Navigator/privateAttribution|privateAttribution}}` attribute.
```

Comment:
This link doesn't seem to be rendering correctly, probably due to the surrounding backticks.

Reply:
It might work if the backticks were replaced with `<code>...</code>`.

Reply:
I should have checked, sorry.  Fixed.

Reply:
Looks like it's still being rendered as `{{Navigator|navigator}}.{{Navigator/privateAttribution|privateAttribution}}`.

Reply:
Hmm, I ran a local build and it renders neatly.

![image](https://github.com/user-attachments/assets/ec59c01d-47c9-41e9-a70d-d90858ff01d3)

Maybe the preview tool is not updating correctly, or maybe the version of bikeshed it is using is different.  I have no idea what is going on.